.circleci/test.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+##
+## Test that GovCMS scaffold can be built using Satis.
+##
+
+# shellcheck disable=SC2002,SC2015
+
+# Satis application directory.
+APP_DIR="${APP_DIR:-./app}"
+
+# The branch of Satis to test against.
+SATIS_BRANCH="${SATIS_BRANCH:-}"
+
+# Directory where scaffold is installed into.
+GOVCMS_SCAFFOLD_DIR="${GOVCMS_SCAFFOLD_DIR:-/tmp/govcms-build}"
+
+#-------------------------------------------------------------------------------
+
+echo "--> Starting Satis web server on http://localhost:4141."
```

Comment:
this part allows to kill previously started inbuilt php server from the previous run, start the server, wait for app to be ready and  assess that the server was successfully started